CONTRIBUTING.md

```diff
@@ -1,60 +1,81 @@
 # Contributing to the Opendatahub Operator
 
-Thanks for your interest in this project. You can contribute to this project in many different ways.
+Thanks for your interest in the opendatahub-operator project! You can contribute to this project in various ways: filing bug reports, proposing features, submitting pull requests (PRs), and improving documentation.
 
-## Issues and Enhancements
+Before you begin, please take a look at our contribution guidelines below to ensure that your contribuutions are aligned with the project's goals.
 
-Please let us know via our GitHub issue tracker if you find a problem, even if you don't have a fix for it.
-The ideal issue report should be descriptive, and where possible include the steps we can take to reproduce the problem for ourselves.
+## Reporting Issues
+Issues are tracked using [Jira](https://issues.redhat.com/secure/RapidBoard.jspa?rapidView=18680#). If you encounter a bug or have suggestions for enhancements, please follow the steps below:
 
-Please go to [issue tracker](https://github.com/opendatahub-io/opendatahub-operator/issues) and create a "New issue".
-Choose suitable issue type and fill in description as detailed as possible.
-Lastly, add extra label for this issue if that can help us refine it.
+1. **Check for Existing Issues:** Before creating a new issue, search the Jira project to see if a similar issue already exists.
+2. **Create a Jira Ticket:** If the issue doesn’t exist, create a new ticket in Jira. 
+   - **For Feature Requests:**  Set the issue type to be `Initative`
```

Comment:
I agree with @adelton that Initiative is internally used differently, so I would propose Story/Task, doesn't really matter, but we shouldn't go 2 levels higher (Initiative)